client/src/app/__tests__/EmptyTabSpec.js

```diff
@@ -30,13 +27,13 @@ describe('<EmptyTab>', function() {
 
       // given
       const onAction = sinon.spy();
-      const {
-        tree
-      } = createEmptyTab({ onAction });
-      const buttons = tree.find('button');
+
+      createEmptyTab({ onAction });
```

Comment:
Could be this, cf. https://github.com/camunda/camunda-modeler/pull/5148.

```suggestion
      const { 
        getAllByRole
      } = createEmptyTab({ onAction });
```